examples/kubernetes/kustomization.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - namespace.yaml
+  - deployment.yaml
+  - service.yaml
+
+# If you want to use the grafana-agent, uncomment below.
```

Comment:
not relevant

Reply:
Fair enough, I'll remove it.

Reply:
Removed in https://github.com/jaegertracing/jaeger/pull/3645/commits/d09add6e1d564c03e471d2f0d5a1841684cc72a7